src/gui_boxmenu.c

```diff
@@ -266,6 +270,22 @@ long gfa_controlled_creature_has_instance(struct GuiBox *gbox, struct GuiBoxOpti
     return creature_instance_is_available(thing, *tag);
 }
 
+long gf_all_doors(struct GuiBox *gbox, struct GuiBoxOption *goptn, unsigned char btn, long *tag)
+{
+    struct PlayerInfo* player = get_my_player();
+    //  if (player->cheat_mode == 0) return false; -- there's no cheat_mode flag yet
```

Comment:
What's this about? Is this a `//todo`?

Reply:
It seems to be. I just adapted this from gf_research_magic. The other cheats also have this comment.